handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslServerContext.java

```diff
@@ -51,6 +52,9 @@
     private static final byte[] ID = {'n', 'e', 't', 't', 'y'};
     private final OpenSslServerSessionContext sessionContext;
 
+    private static final boolean ENABLE_SESSION_TICKET =
+            SystemPropertyUtil.getBoolean("jdk.tls.server.enableSessionTicketExtension", false);
```

Comment:
FYI: JDK provider also has `jdk.tls.server.sessionTicketTimeout` option (default to 24 hours) [1], but IIUC BoringSSL does not provide API to change default rotation interval and always uses 48 hours [2].

[1] https://docs.oracle.com/en/java/javase/11/security/java-secure-socket-extension-jsse-reference-guide.html#GUID-A41282C3-19A3-400A-A40F-86F4DA22ABA9
[2] https://commondatastorage.googleapis.com/chromium-boringssl-docs/ssl.h.html#SSL_DEFAULT_TICKET_KEY_ROTATION_INTERVAL

Reply:
yep there is no API To change this in BoringSSL.